automation/taskcluster/decision_task.py

```diff
@@ -110,6 +124,37 @@ def create_variant_assemble_task(variant):
     )
 
 
+def create_dep_signing_task(artifacts):
+    return create_raw_task(
+        name='dep-sign',
+        description='Dep-signing for performance testing',
+        workerType='mobile-signing-dep-v1',
+        scopes=[
+            "project:mobile:reference-browser:releng:signing:format:autograph_apk_reference_browser",
+            "project:mobile:reference-browser:releng:signing:cert:{}".format('dep-signing')
+        ],
+        treeherder={
+            'jobKind': 'other',
+            'machine': {
+              'platform': 'android-all',
+            },
+            'symbol': 'Ns',
+            'tier': 1,
+        },
+        payload={
+            "maxRunTime": 3600,
+            "upstreamArtifacts": [{
+                'formats': ['autograph_apk_reference_browser'],
+                'paths': [apk_path],
+                'taskId': task_id,
+                'taskType': 'build'
+            } for (apk_path, task_id) in artifacts]
```

Comment:
Chatted on Slack: let's have 1 task matching 1 build, instead of 1 task depending on all builds. In the specific case of raptor: that allows to flatten the workload => instead of having `$n` raptor tasks started at once, you get them one by one